packages/react-components/react-nav-preview/src/components/Nav/useNav.ts

```diff
@@ -83,5 +92,62 @@ export const useNav_unstable = (props: NavProps, ref: React.Ref<HTMLDivElement>)
     onUnregister,
     onSelect,
     getRegisteredNavItems,
+    onRequestNavCategoryItemToggle,
+    openItems,
   };
 };
+
+// /**
+//  * Initial value for the uncontrolled case of the list of open indexes
+//  */
+// function initializeUncontrolledOpenItems({ defaultOpenItems }: Pick<NavProps, 'defaultOpenItems'>): NavItemValue[] {
+//   if (defaultOpenItems !== undefined) {
+//     if (Array.isArray(defaultOpenItems)) {
+//       return [defaultOpenItems[0]];
+//     }
+//     return [defaultOpenItems];
+//   }
+//   return [];
+// }
+
+// /**
+//  * Normalizes Accordion index into an array of indexes
+//  */
+// function normalizeValues(index?: NavItemValue | NavItemValue[]): NavItemValue[] | undefined {
+//   if (index === undefined) {
+//     return undefined;
+//   }
+//   return Array.isArray(index) ? index : [index];
+// }
+
+// temp implementation of the above function.
+function normalizeValues(index?: NavItemValue | NavItemValue[]): NavItemValue[] | undefined {
+  return undefined;
+}
+
+/**
+ * Updates the list of open indexes based on an index that changes
+ * @param value - the index that will change
+ * @param previousOpenItems - list of current open indexes
+ * @param multiple - if Nav support multiple SubItemGroups opened at the same time
+ * @param collapsible - if Nav support multiple SubItemGroups closed at the same time
+ */
+function updateOpenItems(
+  value: NavItemValue,
+  previousOpenItems: NavItemValue[],
+  multiple: boolean,
+  collapsible: boolean,
+) {
```

Comment:
We are starting to adopt option bags for functions/hooks. This is a minor nit, but if you think the params won't change it's fine to keep as is since it's not exported

```suggestion
function updateOpenItems({
  value,
  previousOpenItems,
  multiple,
  collapsible
  }: {
  value: NavItemValue,
  previousOpenItems: NavItemValue[],
  multiple: boolean,
  collapsible: boolean,
}) {
```

Reply:
Cool - I understand the change in pattern - but I'm curious how the extra boilerplate is advantageous to straight params?

Reply:
Offline recap - we chatted about this. Good pattern suggestion, but we'll adjust when it comes time to expose an API.